src/Plugin/Field/FieldFormatter/Strawberry3DFormatter.php

```diff
@@ -96,25 +78,16 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
         '#field_suffix' => $this->t('pixels'),
         '#min' => 0,
       ],
-    ];
+    ] + parent::settingsForm($form, $form_state);
   }
 
   /**
    * {@inheritdoc}
    */
   public function settingsSummary() {
-    $summary = [];
+    $summary = parent::settingsSummary();
```

Comment:
@DiegoPino answer is no, it is a small point, so i will not feel strongly, but I chose this way because the UI is less jarring when it changes in the dropdown this way.